enhancements/kubelet/user-namespaces-support.md

```diff
@@ -0,0 +1,454 @@
+| FSGroup                         | fsGroup                     | If a pod is in a user namespace, this field can safely be RunAsAny, however for the most restrictive policies, continuing to enforce this would add an additional layer of security.                                      |
+| ReadOnlyRootFilesystem          | readOnlyRootFilesystem      | No impact--the handling of read only rootfs should not change if a pod is in a user namespace.                                                                                                                            |
+| Users                           | N/A                         | No impact--this field is dictating which openshift user can access this, and is not related to a pod field.                                                                                                               |
+| Groups                          | N/A                         | No impact--this field is dictating which openshift user can access this, and is not related to a pod field.                                                                                                               |
+| SeccompProfiles                 | seccompProfile              | No impact if a pod is in a user namespace on this field, but if this field is set to `*` or "empty" then it may be tempting to allow all capabilities, but the privileged escalation piece is another aspect to consider. |
+| AllowedUnsafeSysctls            | sysctls                     | No impact--the handling of sysctls should not change if a pod is in a user namespace.                                                                                                                                     |
+| ForbiddenSysctls                | sysctls                     | No impact--the handling of sysctls should not change if a pod is in a user namespace.                                                                                                                                     |
+
+##### `UserNamespaceLevel` field
+
+Add the following to the [Openshift API](https://github.com/openshift/api/blob/76a71dac36a08eab1b240c6c8d4e39c813b1b12b/security/v1/types.go):
+
+```
+       // userNamespaceLevel determines if the policy allows host users in containers.
+       // Valid values are "AllowHostLevel", "RequirePodLevel", and omitted.
+       // When "AllowHostLevel" is set, a pod author may set `hostUsers` to either `true` or `false`.
+       // When "RequirePodLevel" is set, a pod author must `hostUsers` to `false`.
+       // When omitted, this means no opinion and the platform is left to choose a reasonable default,
+       // which is subject to change over time.
+       // The current default is "AllowHostLevel".
+       // +openshift:enable:FeatureGate=UserNamespacesPodSecurityStandards
```

Comment:
You sure you want to couple these together?  The SCC could be done distinctly I think.

Reply:
I think they're logically tied together so I think so. Why would i not?